srcpkgs/ruby-build/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'ruby-build'
 pkgname=ruby-build
-version=20221124
+version=20230919
 revision=1
 depends="bash"
 short_desc="Compile and install Ruby"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Moe Derballa <mderballa@hotmail.com>"
```

Comment:
Please remove the maintainer change. [We prefer that people demonstrate a history of stewardship before adopting templates.](https://github.com/void-linux/void-packages/blob/7d0d5b89af0301e7db27533c95b2183ac2035057/CONTRIBUTING.md#adopting-a-template)